### DISCUSSION TITLE
what does "next proxy index: 0 / Count: 0" mean?

### DISCUSSION THREAD

**Discussion by Jamjano** [jef/streetmerchant#1739] [General]
Bot runs nicely on my private PC, but when running on my PC from workplace it outputs this every few lines:

![image](https://user-images.githubusercontent.com/64227162/104974411-358b1180-59f8-11eb-9116-6a45f13b018d.png)

or sth. like: 

![image](https://user-images.githubusercontent.com/64227162/104974432-4a67a500-59f8-11eb-848f-d92e1f77f881.png)

No real issue and not sure whether this increases latency, but would like to understand it - anyone having a hint?

I don't have any proxy set, and settings are standard except for stores and models.

`PROXY_ADDRESS=`
`PROXY_PROTOCOL=`
`PROXY_PORT=`

**Comment by wofnull** [comment]
Could it be that you are having a system wide proxy set up on your workplace system? SM uses the system provided proxy first even if no proxy in config is set to prevent the script from failing. The messages do come from this.

**Reply by Jamjano** [reply]
Okay, yes makes sense. Thanks!
